main/hw-esp32.c

```diff
@@ -5,6 +5,8 @@
 #include "esp_private/periph_ctrl.h"
 #include "esp_timer.h"
 #include "driver/uart.h"
+#include "soc/uart_periph.h"
```

Comment:
did you get a compilation error without this? which IDF version?

Reply:
I am using esp-idf 5.2, the master branch.